docs/3.2/kubernetes_ssh.md

```diff
@@ -0,0 +1,220 @@
+# Kubernetes and SSH Integration Guide
+
+Teleport v.3.0+ has the ability to act as a compliance gateway for managing privileged access to Kubernetes clusters. This enables the following capabilities:
+
+* A Teleport Proxy can act as a single authentication endpoint for both SSH and
+  Kubernetes. Users can authenticate against a Teleport proxy using Teleport's `tsh login` command and retrieve credentials for both SSH and Kubernetes API.
+* Users RBAC roles are always synchronized between SSH and Kubernetes, making
+  it easier to implement policies like _developers must not access production
+  data_.
+* Teleport's session recording and audit log extend to Kubernetes, as well.
+  Regular `kubectl exec` commands are logged into the audit log and the
+  interactive commands are recorded as regular sessions that can be stored and replayed in the
+  future.
+
+![ssh-kubernetes-integration](img/teleport-kube.png)
+
+This guide will walk you through the steps required to configure Teleport to
+work as a unified gateway for both SSH and Kubernetes. We will cover both the open
+source and enterprise editions of Teleport.
+
+For this guide, we'll be using an instance of Kubernetes running on [Google's GKE](https://cloud.google.com/kubernetes-engine/)
+but this guide should apply with any upstream Kubernetes instance.
+
+## Teleport Proxy Service
```

Comment:
Changes for 3.2 are from lines 24-110 - please validate that these details are correct.